core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -241,10 +241,6 @@ class EMConfig {
                     "collecting heuristics with 'heuristicsForSQL'")
         }
 
```

Comment:
did u rebase agaisnt the most recent version? looks like here you modified a file, Man did modifications on it, and you either didn't pull that version, or didn''t merge her changes